sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/DataLakeFileSystemAsyncClient.java

```diff
@@ -73,6 +73,8 @@
 
     public static final String ROOT_FILESYSTEM_NAME = "$root";
 
+    public static final String ROOT_DIRECTORY_NAME = "";
```

Comment:
If these are both staying in public API we should add Javadocs that explain their purpose.

Reply:
I'll make them private